session/bootstrap.go

```diff
@@ -791,6 +800,11 @@ func upgrade(s Session) {
 	updateBootstrapVer(s)
 	_, err = s.ExecuteInternal(context.Background(), "COMMIT")
 
+	if err == nil && ver <= version92 {
+		logutil.BgLogger().Info("start migrate DDLs")
+		err = domain.GetDomain(s).DDL().MigrateExistingDDLs()
```

Comment:
Should it check `variable.EnableConcurrentDDL` before migrating?

Reply:
no, check the ver is under version92 is enough. it only happens in rolling upgrade